plugins/Live/angularjs/live-widget-refresh/live-widget-refresh.directive.js

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Piwik - free/libre analytics platform
```

Comment:
Should this be updated to Matomo branding?

Reply:
we'll change this at some stage on all files in a new PR. so far we kept it for existing files.